### PR TITLE
Update Sidekiq migrate

### PR DESCRIPTION
**Pull request by izakp**
As discussed in https://github.com/artsy/docker-images/pull/57 remove the `ACTUAL_RUN` and really run by default - replace with `DRY_RUN` env var.

Need  a sanity check on the changes to conditional statements!  `unless` continues to confound me.

Also migrate stats by default because it's so easy and simplifies things.

### CODE REVIEW

File: sidekiq-migrate/migrate.rb [hunk 1]
```diff
@@ -21,7 +20,7 @@
   moved_jobs_counter = Hash.new(0)
   rqn = "queue:#{q}"
 
-  if actual_run and clean_up
+  unless dry_run and clean_up
```
Comment (mzikherman) [inline sidekiq-migrate/migrate.rb]: Demorgan, should be `unless dry_run or !clean_up`.

I'm with you on compound conditions with `unless` being counter intuitive sometimes. Maybe clearer with `if !dry_run && clean_up`?

Comment (izakp) [reply sidekiq-migrate/migrate.rb]: Thanks for the formal logic check!  I like `if !dry_run && clean_up` - it's more explicit

File: sidekiq-migrate/migrate.rb [hunk 2]
```diff
@@ -53,27 +52,25 @@
 
   set_jobs.each do |job_json, run_at|
     puts job_json if debug
-    new_redis.zadd(set_type, run_at, job_json) if actual_run
-    old_redis.zrem(set_type, job_json) if actual_run and clean_up
-    moved_jobs_counter[JSON.parse(job_json)['class']] += 1 if actual_run
+    new_redis.zadd(set_type, run_at, job_json) unless dry_run
+    old_redis.zrem(set_type, job_json) unless dry_run or not clean_up
```
Comment (izakp) [inline sidekiq-migrate/migrate.rb]: Similarly here... `if !dry_run && clean_up`

File: sidekiq-migrate/migrate.rb [hunk 3]
```diff
@@ -53,27 +52,25 @@
...
-    old_redis.del(k) if actual_run and clean_up
-  end
-  puts "Migrated #{stats.length} stats"
+puts "\n\nMigrating stats..."
+stats = old_redis.keys("stat:*")
+stats.each do |k|
+  v = old_redis.get(k)
+  puts "#{k} #{v}" if debug
+  new_redis.set(k, v) unless dry_run
+  old_redis.del(k) unless dry_run or not clean_up
```
Comment (izakp) [inline sidekiq-migrate/migrate.rb]: And here

**Pull request merged**